src/test/codegen/some-abis-do-extend-params-to-32-bits.rs

```diff
@@ -0,0 +1,204 @@
+// compile-flags: -Cno-prepopulate-passes
+
+// revisions:x86_64 i686 aarch64-apple aarch64-windows aarch64-linux arm riscv
+
+//[x86_64] compile-flags: --target x86_64-unknown-uefi
+//[x86_64] needs-llvm-components:
+//[i686] compile-flags: --target i686-unknown-linux-musl
+//[i686] needs-llvm-components: i686
```

Comment:
I believe the LLVM component is called `x86` for both of the above targets.

Reply:
Correct, as seen here:
https://github.com/rust-lang/rust/commit/493fe8008b33c576240bfa6979760e0e5482d5af

Reply:
Incorporated into https://github.com/rust-lang/rust/pull/97800/commits/7913b8543e90602e98092f23be31259841671ff1